src/PartielsPy/partiels.py

```diff
@@ -21,6 +24,13 @@ class Partiels:
     version and a warning is trigger if not matching.
     """
 
+    class DefaultTemplateEnum(Enum):
```

Comment:
The problem I see with this enum is that if we add templates in the future, we have to modify the Python code. I think it's better to have a function that returns the path to the templates folder (this function shouldn't even have to be in the class) or to do nothing at all and let people define the full path of the templates (but we can talk about it in the ReadMe and/or the documentation).